Projekty/Projekt1/PatrykTomaszewskiMateuszZiemla/Milestone 2.ipynb

```diff
@@ -0,0 +1,503 @@
+{
```

Comment:
<p>Fajen, ale zacznijmy od base line'u.</p><p> </p><p>Jeśli regresja logistyczna daje 0.96, to jaki zysk z waszej sieci?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Moglibyście jasno przedstawić jakie jest acc dla jakiego modelu</p>